stuffioAPI/stuffioAPI/stuffioAPI.csproj.user

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
We don't care about your user settings.

Ignore this file

Reply:
Done